resources/function_help/json/is_selected

```diff
@@ -14,9 +14,9 @@
             "variant": "One 'feature' parameter",
             "variant_description": "If called with a 'feature' parameter only, the function returns true if the specified feature from the current layer is selected.",
             "arguments": [ { "arg": "feature", "description": "The feature which should be checked for selection." } ],
-            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if the current atlas feature is selected." },
-                          { "expression": "is_selected(get_feature('streets', 'name', 'Main St.')))", "returns": "True if the unique named \"Main St.\" feature on the active \"streets\" layer is selected." },
-                          { "expression": "is_selected(get_feature_by_id('streets', 1))", "returns": "True if the feature with the id 1 on the active \"streets\" layer is selected." }
+            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if a selected feature on the current layer is the active atlas feature." },
+                          { "expression": "is_selected(get_feature(@layer, 'name', 'Main St.')))", "returns": "True if the unique named \"Main St.\" feature on the current layer is selected." },
+                          { "expression": "is_selected(get_feature_by_id(@layer, 1))", "returns": "True if the feature with the id 1 on the current layer is selected." }
```

Comment:
```suggestion
                          { "expression": "is_selected(get_feature_by_id(@layer, 1))", "returns": "True if the feature with the id 1 is selected." }
```

I wonder if we shouldn't just shorten this. The "on the current layer" part seems to be too obvious to even mention ?

Reply:
I don't know whether it's too obvious (I'm still unsure I understand the logic behind the function) but the ideas were to somehow keep the same structure for this group of examples, and ease comparison with the next group of expressions (in variant 3)

Reply:
It was the term "current layer" which made me unsure. For me it's automatically triggering an association to "active layer" (`iface.activeLayer()`) which is incorrect.
But reading through the whole page again, I can see that in context it does make sense, so let's forget about it.
Good as is :+1: